script.lazytv/changelog.txt

```diff
@@ -0,0 +1,75 @@
+0.9.92
```

Comment:
can you not include the changelog with the submission? this is no longer parsed by kodi in favor of the `<news>` tag in addon.xml